rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -39,14 +39,14 @@ def coco(annotations_file='', ltrb=True, masks=False, ratio=False,
     return (meta_data, labels, bboxes)
 
 
-def file(file_root, file_filters=None):
+def file(file_root, file_list='', file_filters=None):
     Pipeline._current_pipeline._reader = "labelReader"
     # Output
     labels = []
     if file_filters is None:
         file_filters = ["*.jpg", "*.jpeg", "*.png", "*.bmp", "*.tif", "*.tiff", "*.pnm", "*.ppm", "*.pgm", "*.pbm", "*.jp2", "*.webp"]
```

Comment:
Are u planning on using file function itself for reading the audio files? If so, please add the audio file extensions to the file_filters list

Reply:
Why I am seeing that the file_filters is not used anywhere after assignment ?
May I know whats the purpose of this list